integration-tests/README.md

```diff
@@ -66,7 +66,40 @@ Integration tests can also be run with either Java 8 or Java 11 by adding -Djvm.
 can either be 8 or 11.
 
 Druid's configuration (using Docker) can be overrided by providing -Doverride.config.path=<PATH_TO_FILE>. 
-The file must contain one property per line, the key must start with druid_ and the format should be snake case. 
+The file must contain one property per line, the key must start with `druid_` and the format should be snake case. 
+
+Running Tests Using A Quickstart Cluster
+-------------------
+
+When writing integration tests, it can be helpful to test against a quickstart
+cluster so that you can set up remote debugging with in your developer
+environment. This section walks you through setting up the integration tests
+so that it can run against a quickstart cluster running on your development
+machine.
+
+Note that not all features run by default on a quickstart cluster, so it may
```

Comment:
@maytasm3 I was worried about what you mentioned as a top level comment on this PR too. I tried to talk about it here, but maybe the warning is not big and scary enough. Any suggestions on how to make it more prominent? Also I wonder if I should point to the docker setup scripts for the next person who might find a test that fails when run against quickstart.

Reply:
Other places in the docs use a blockquote like: https://druid.apache.org/docs/latest/querying/lookups.html

Reply:
I feel the term "feature" might be a little misleading. Some of the setup on the docker cluster are just pre-inserted data into the metadata store or certain files place at certain locations. The feature itself like pulling segments from pathSpec in metadatastore is always there. I think as we add more and more IT and/or setup scripts, it will also be hard to knows which can work out of the box on quickstart vs. which can only work on docker cluster. I think it might help if we can list out which groups can work out of the box on quickstart vs. which can only work on docker cluster in the README. In the file with the group definition (TestNGGroup) we can then list out special setup needed (if needed) for those groups.

Reply:
👍 Added a note on the setup, but it's kinda hard to know which groups depend on what right now, maybe after using this for a few months the test group will be more complete. I added instructions to this file and the group so that other devs can add more tests to this list.